test/test_helper.rb

```diff
@@ -62,4 +62,5 @@ class Spec
 require 'closure_tree'
 require_relative '../spec/support/schema'
 require_relative '../spec/support/models'
+require_relative '../spec/support/helpers'
```

Comment:
This helper is for rspec. 
Minitest will need a assertion method instead, ex : assert_database_queries_count(expected_count, %block)